eduvarjaor/talk-text
# Integrate server with client

## Description

The Firebase function was integrated with the client, and the user interface was improved.

## **Related Issue:**

N/A

## **Main changes explained:**

- Configured Firebase Function to return Openai response to client
- Configured client to receive the backend response
- Improved client user interface

## **How to test:**

1. check into current branch
2. cd talk-text/client
3. do `npm install` to run this PR locally
4.  run `npm run dev`

## **Screenshots or videos of changes:**

If the user interface looks like the photo and the app is receiving properly the Openai response, everything worked as intended.

![Screenshot_1](https://github.com/eduvarjaor/talk-text/assets/86692035/90341634-8d58-43d7-99ce-e15defcecf4d)
